### PR TITLE
v1 VSA: Update policy levels to indicate the track

### LINKED ISSUE #725: v1 VSA: Update policy levels to indicate the track
In VSA https://slsa.dev/verification_summary/v1

The policy levels don't indicate the track: `build`, `source`. It would be useful to add the track information. Possible options:

### Add a nested track field
```yaml
"policy_levels": {
  "build": "SLSA_LEVEL_3",
  "source": "SLSA_LEVEL_3",
  ...
}
```

### Add track in policy_level
```yaml
"policy_source_level": "SLSA_LEVEL_3",
"policy_build_level": "SLSA_LEVEL_3",
```

### Same as above + rename level
```yaml
"policy_source_level": "SLSA_SOURCE_LEVEL_3",
"policy_build_level": "SLSA_BUILD_LEVEL_3",
```

### PR DESCRIPTION
**Pull request by kpk47**
The VSA format doesn't account for different SLSA tracks (i.e. it tracks SLSA_LEVEL_3 instead of SLSA_BUILD_LEVEL_3). This change adds tracks to the VSA format for both the artifact itself and its dependencies.

Fixes #725

### CODE REVIEW

File: docs/verification_summary/v1.md [hunk 1]
```diff
@@ -83,7 +83,9 @@ to establish minimum requirements on dependencies SLSA levels may use
     ...
   ],
   "verification_result": "<PASSED|FAILED>",
-  "policy_level": "<SlsaResult>",
+  "policy_level": {
```
Comment (laurentsimon) [inline docs/verification_summary/v1.md:86]: nit: `policy_levels`? There is one policy per track.

Comment (kpk47) [reply docs/verification_summary/v1.md]: Done

File: docs/verification_summary/v1.md [hunk 2]
```diff
@@ -83,7 +83,9 @@ to establish minimum requirements on dependencies SLSA levels may use
     ...
   ],
   "verification_result": "<PASSED|FAILED>",
-  "policy_level": "<SlsaResult>",
+  "policy_levels": {
+    "build": "<SlsaResult>"
+  },
   "dependency_levels": {
     "<SlsaResult>": <Int>,
```
Comment (laurentsimon) [inline docs/verification_summary/v1.md:90]: I think this must be changed too. Probably you should declare a new Object:
```yaml
policy_level {
  track_name: SlsaResult
}

policy_levels: []policy_level,
dependency_levels: []policy_level,
```

File: docs/verification_summary/v1.md [hunk 3]
```diff
@@ -192,11 +190,10 @@ of the other top-level fields, such as `subject`, see [Statement]._
 > Either “PASSED” or “FAILED” to indicate if the artifact passed or failed the policy verification.
 
 <a id="policy_level"></a>
-`policy_level` _object ([SlsaResult]), required_
+`policy_level` _string ([SlsaResult]), repeated_
```
Comment (TomHennen) [inline docs/verification_summary/v1.md:193]: Would it make sense to rename this field to 'levels' ?

Comment (MarkLodato) [reply docs/verification_summary/v1.md:193]: or `verified_level` to be a bit more verbose

File: docs/verification_summary/v1.md [hunk 4]
```diff
@@ -233,11 +234,11 @@ WARNING: This is just for demonstration purposes.
     }
   ],
   "verification_result": "PASSED",
-  "policy_level": "SLSA_LEVEL_3",
+  "policy_level": "SLSA_BUILD_LEVEL_3",
```
Comment (MarkLodato) [inline docs/verification_summary/v1.md:237]:
```suggestion
  "policy_level": ["SLSA_BUILD_LEVEL_3"],
```

File: docs/verification_summary/v1.md [hunk 5]
```diff
@@ -190,9 +190,10 @@ of the other top-level fields, such as `subject`, see [Statement]._
 > Either “PASSED” or “FAILED” to indicate if the artifact passed or failed the policy verification.
 
 <a id="policy_level"></a>
-`policy_level` _string ([SlsaResult]), required_
+`policy_level` _string ([SlsaResult]), repeated_
```
Comment (MarkLodato) [inline docs/verification_summary/v1.md:193]:
```suggestion
`policy_level` _array ([SlsaResult]), optional_
```

Also need to update the schema above to make this an array.

File: docs/verification_summary/v1.md (hunk 3 above)
Comment (TomHennen) [reply docs/verification_summary/v1.md:193]: nit: if it's repeated should it be plural :)

**Review by MarkLodato — CHANGES_REQUESTED** [review]

File: docs/verification_summary/v1.md [hunk 6]
```diff
@@ -190,9 +190,10 @@ of the other top-level fields, such as `subject`, see [Statement]._
...
 
-> Indicates what SLSA level the artifact itself (and not its dependencies) was verified at or "FAILED" if policy verification failed.
+> Indicates which SLSA tracks and levels the artifact itself (and not its
+> dependencies) were verified at or "FAILED" if policy verification failed.
```
[the remark this answers is not in the document]
Comment (kpk47) [reply docs/verification_summary/v1.md]: Done

File: docs/verification_summary/v1.md (hunk 5 above)
Comment (kpk47) [reply docs/verification_summary/v1.md]: Done

File: docs/verification_summary/v1.md (hunk 4 above)
Comment (kpk47) [reply docs/verification_summary/v1.md]: Done

**Review by MarkLodato — APPROVED** [review]

**Pull request merged**